### PR TITLE
Add postgresql-debversion extension to katello

### PR DESCRIPTION
**Pull request by m-bucher**
educated copy of https://github.com/theforeman/puppet-katello/pull/332 for the `debversion`-extension

### CODE REVIEW

File: manifests/application.pp [hunk 1]
```diff
@@ -71,6 +72,9 @@
   }
 
   if $manage_db {
+    package { $postgresql_debversion_package:
+      ensure => installed,
+    }
```
Comment (ekohl) [inline manifests/application.pp:77]: I think this should only be done if `$enable_deb` is true, right? Or will it be needed unconditionally?

Comment (m-bucher) [reply manifests/application.pp:77]: I think we need it unconditionally, because the database-migration in katello will also run unconditionally.

Comment (jlsherrill) [reply manifests/application.pp:77]: yes, whether a content type is enabled/installed doesn't change the schema that is deployed, so its sitll needed

**Comment by ekohl** [comment]
Revert PR: https://github.com/theforeman/puppet-katello/pull/353

**Pull request merged**